can/io/asc.py

```diff
@@ -49,10 +47,18 @@ def _extract_can_id(str_can_id):
         return can_id, is_extended
 
     def __iter__(self):
+        start_time = 0.0
         for line in self.file:
             # logger.debug("ASCReader: parsing line: '%s'", line.splitlines()[0])
 
             temp = line.strip()
+            
+            #check for timestamp
+            if "begin triggerblock" in temp.lower():
+                _, _, start_time = temp.split(None, 2)
+                start_time = datetime.strptime(start_time, "%a %b %m %I:%M:%S.%f %p %Y").timestamp()
```

Comment:
What happens if the parsing fails? Should we catch some exception?

Reply:
Both of these lines could raise ValueError, if `temp` contains fewer than two whitespaces, or if it does not match the specified date format.

Reply:
One idea would be to raise an error here, and maybe let the user suppress it via a param to the constructor.

Reply:
I think we could fall back to relative timestamps since absolute timestamps is not guaranteed in either log files or buses.

Reply:
> I think we could fall back to relative timestamps since absolute timestamps is not guaranteed in either log files or buses.

Yeah, but maybe print a warning along the way.